packages/govuk-frontend/src/govuk/components/accordion/accordion.yaml

```
@@ -48,30 +48,45 @@ params:
     required: true
     description: An array of sections within the accordion.
     params:
-      - name: heading.text
-        type: string
+      - name: heading
+        type: object
         required: true
-        description: The title of each section. If `heading.html` is supplied, this is ignored.
-      - name: heading.html
-        type: string
-        required: true
-        description: The HTML content of the header for each section. Used as the title for each section. The header is inside the HTML `<button>` element, so you can only add [phrasing content](https://html.spec.whatwg.org/#phrasing-content) to it.
-      - name: summary.text
-        type: string
-        required: false
-        description: Text content for summary line. If `summary.html` is supplied, this is ignored.
-      - name: summary.html
-        type: string
+        description: The heading of each section.
+        params:
+          - name: text
+            type: string
+            required: true
+            description: If `html` is set, this is not required. The heading text of each section. If `html` is provided, the `text` option will be ignored.
```

Worth merging the two related clauses?

```suggestion
            description: The heading text of each section. If `html` is provided, this option is not required and will be ignored.
```

---

@36degrees I've picked the most common flavour of description (same as @querkmachine in [#3878](https://github.com/alphagov/govuk-frontend/pull/3878/files))

Can either delay this PR and fix all the others, or merge this (consistently repetitive) and improve later?

---

Eurgh, yeah, OK, let's improve it later.